### DISCUSSION TITLE
What happened to the responsive grid

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#2742] [Q&A] [answered]
Hey everyone,

I just came back to reflex after a while to start a new project and cannot seem to find the responsive grid any more. 
Is it just me or is there no such component any more? Any alternatives?

I just wish to have components stacked in a certain way on computer screen and another way on mobile.

Thanks in advance :)

**Comment by picklelo** [comment, marked as answer]
@NanoscaleSimulations the Chakra components are still available in the `rx.chakra` namespace: https://reflex.dev/docs/library/chakra/layout/responsivegrid/

**Reply by NanoscaleSimulations** [reply]
Ah I wasnt even aware this was a chakra component. Thank you.
However, I tried to use the search bar on the reflex website to search for responsive grid but it didn't appear at all. Possibly a bug on the website?
